### PR TITLE
[zh]Resync cluster-administration networking file

### PR DESCRIPTION
**Pull request by mengjiao-liu**
[zh]Resync cluster-administration networking file

### CODE REVIEW

File: content/zh/docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -49,80 +51,7 @@ Kubernetes 的宗旨就是在应用之间共享机器。
...
-implemented is also a detail of the container runtime. The `Pod` itself is
-blind to the existence or non-existence of host ports.
--->
-如何实现这一点是正在使用的容器运行时的特定信息。
-
-也可以在 `node` 本身通过端口去请求你的 `Pod`（称之为主机端口），
-但这是一个很特殊的操作。转发方式如何实现也是容器运行时的细节。
-`Pod` 自己并不知道这些主机端口是否存在。
-
+要了解 Kubernetes 网络模型，请参阅 [此处](/docs/concepts/services-networking/)。
```
Comment (tengqm) [inline content/zh/docs/concepts/cluster-administration/networking.md:54]:
```suggestion
要了解 Kubernetes 网络模型，请参阅[此处](/zh/docs/concepts/services-networking/)。
```

Comment (mengjiao-liu) [reply content/zh/docs/concepts/cluster-administration/networking.md]: Updated.Thanks!

**Pull request merged**